java/client/src/main/java/glide/api/commands/SetBaseCommands.java

```diff
@@ -75,4 +75,19 @@ public interface SetBaseCommands {
      * }</pre>
      */
     CompletableFuture<Long> scard(String key);
+
+    /**
+     * Gets the intersection of all the given sets.<br>
+     * Missing set is interpreted as empty and causes an empty response.
```

Comment:
I think you could remove this line and add it  to the return.
     * If any set is missing it is interpreted as empty and causes an empty response.

Reply:
Fixed thanks